client/extensions/woocommerce/app/settings/payments/payment-method-paypal.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormLegend from 'components/forms/form-legend';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import PaymentMethodEditFormToggle from './payment-method-edit-form-toggle';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+class PaymentMethodPaypal extends Component {
+
+	static propTypes = {
+		method: PropTypes.shape( {
+			settings: PropTypes.shape( {
+				title: PropTypes.shape( {
+					id: PropTypes.string.isRequired,
+					label: PropTypes.string.isRequired,
+					type: PropTypes.string.isRequired,
+					value: PropTypes.string.isRequired,
+				} ),
+			} ),
+		} ),
+		translate: PropTypes.func.isRequired,
+		onEditField: PropTypes.func.isRequired,
+	};
+
+	onEditFieldHandler = ( e ) => {
+		this.props.onEditField( e.target.name, e.target.value );
+	}
+
+	onSaveHandler = () => {
+		this.props.onSave( this.props.method );
+	}
+
+	renderEnabledField = ( isEnabled ) => {
+		return (
+			<PaymentMethodEditFormToggle
+				checked={ isEnabled === 'yes' ? true : false }
+				name="enabled"
+				onChange={ this.onEditFieldHandler } />
+		);
+	}
+
+	render() {
+		const { method, translate } = this.props;
+		return (
+			<div className="payments__method-edit-fields">
+				<FormFieldset className="payments__method-edit-field-container">
+					<FormLabel>{ translate( 'Enabled' ) }</FormLabel>
+					{ this.renderEnabledField( method.settings.enabled.value ) }
```

Comment:
Maybe destructure `settings` out of method as well, since it's used a few times.